_data/simple-icons.json

```
@@ -7846,8 +7846,8 @@
         },
         {
             "title": "S7 Airlines",
-            "hex": "BED600",
-            "source": "https://www.s7.ru/"
+            "hex": "C4D600",
+            "source": "https://www.s7.ru/en/info/s7-airlines/brand/"
```

Does it make sense to add this as guidelines as well?

```suggestion
            "source": "https://www.s7.ru/en/info/s7-airlines/brand/",
            "guidelines": "https://www.s7.ru/en/info/s7-airlines/brand/"
```

---

I was quite mitigated about this one, but I think it makes sense!